sql-cli/sql_cli/__main__.py

```diff
@@ -12,7 +12,9 @@
 app = typer.Typer(add_completion=False, context_settings={"help_option_names": ["-h", "--help"]})
 
 
-logging.getLogger("airflow").setLevel(logging.CRITICAL)
+airflow_logger = logging.getLogger("airflow")
```

Comment:
@feluelle did you check out if there would be a way for us to test this behaviour?

Reply:
Good point. Yes, we can.

Reply:
pytest's caplog fixture would be useful

Reply:
https://docs.pytest.org/en/7.1.x/how-to/logging.html

Reply:
Added in ee3da30